converters/toZigbee.js

```diff
@@ -4075,6 +4036,17 @@ const converters = {
                 for (const member of entity.members) {
                     if (member.meta.hasOwnProperty('scenes') && member.meta.scenes.hasOwnProperty(metaKey)) {
                         membersState[member.getDevice().ieeeAddr] = addColorMode(member.meta.scenes[metaKey].state);
+
+                        let recalledState = member.meta.scenes[metaKey].state;
+
+                        // add color_mode if saved state does not contain it
+                        if (!recalledState.hasOwnProperty('color_mode')) {
+                            recalledState = addColorMode(recalledState);
+                        }
+
+                        recalledState = libColor.syncColorState(recalledState, member.meta.state, member.meta.options);
```

Comment:
While testing scene_recall changes, I noticed `member.meta.state` and member.meta.options` are undefined, how would I go about getting this info?

Reply:
It's probably safe to use the group's meta.options, we can't use the groups meta state though... if we do we might leak state from the group to the device for not uniform groups!